pkg/member/member_control.go

```diff
@@ -38,6 +42,9 @@ type Control interface {
 	// AddMemberAsLearner add a new member as a learner to the etcd cluster
 	AddMemberAsLearner(context.Context) error
 
+	// IsClusterScaledup determines whether a etcd cluster is getting scale-up or not.
+	IsClusterScaledup(context.Context, client.Client) (bool, error)
```

Comment:
Scaledup are 2 words and therefore should be camel-cased. So the name should be `IsClusterScaledUp`